Toggl.Foundation.Tests/Sync/States/CleanUp/CheckForInaccessibleWorkspacesTests.cs

```diff
@@ -0,0 +1,74 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Reactive.Linq;
+using System.Threading.Tasks;
+using FluentAssertions;
+using NSubstitute;
+using Toggl.Foundation.DataSources;
+using Toggl.Foundation.Models.Interfaces;
+using Toggl.Foundation.Sync.States.CleanUp;
+using Toggl.Foundation.Tests.Mocks;
+using Toggl.PrimeRadiant;
+using Toggl.PrimeRadiant.Models;
+using Xunit;
+
+namespace Toggl.Foundation.Tests.Sync.States.CleanUp
+{
+    public sealed class CheckForInaccessibleWorkspacesTests
+    {
+        private readonly CheckForInaccessibleWorkspacesState state;
+
+        private readonly ITogglDataSource dataSource = Substitute.For<ITogglDataSource>();
+
+        public CheckForInaccessibleWorkspacesTests()
+        {
+            state = new CheckForInaccessibleWorkspacesState(dataSource);
+        }
+
+        [Theory, LogIfTooSlow]
+        [InlineData(0, SyncStatus.InSync)]
+        [InlineData(1, SyncStatus.InSync)]
+        [InlineData(3, SyncStatus.InSync)]
+        [InlineData(1, SyncStatus.SyncFailed)]
+        [InlineData(3, SyncStatus.SyncFailed)]
+        public async Task ReturnsTheCorrectStateBasedOnTheNumberOfInaccessibleWorkspaces(int numberOfInaccessibleWorkspaces, SyncStatus syncStatus)
+        {
+            List<MockWorkspace> workspaces = new List<MockWorkspace>();
```

Comment:
This doesn't work. You need to remove the `;` at the end of the line. The workspaces defined in the following block won't be added to the list. Also use `var` 😄 
```suggestion
            var workspaces = new List<MockWorkspace>
```